includes/wf_crm_webform_ajax.inc

```diff
@@ -22,7 +24,7 @@ class wf_crm_webform_ajax extends wf_crm_webform_base {
       drupal_json_output(array('' => t('- first choose a country -')));
       exit();
     }
-    drupal_json_output(wf_crm_get_states($input));
+    drupal_json_output(Utils::wf_crm_get_states($input));
```

Comment:
Perhaps this was updated in a search-and-replace? If so, I guess that's fine. However, `drupal_json_output()` isn't a thing in Drupal 8, so if this code actually executes, it'd be a fatal error.

Reply:
That was intended. I fixed the one function call but left this to error for whenever it bubbles up.